TestAssets/TestProjects/DotNetCoreTestProject/Tests.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Can we change the name of this TestProject to be more representative of it's affiliation with VSTest?

For example, something like `VSTestDotNetCoreProject` or `VSTestLibraryProject`